build/npm/navigation-react-native/navigation.react.native.d.ts

```diff
@@ -69,6 +69,10 @@ export interface NavigationBarProps {
      * The color of the title view
      */
     titleColor?: string;
+    /**
+     * The title for the back button when this scene is showing
```

Comment:
Change the comment to "The title for the back button" because there's no need for the "when scene is showing"